shared/ui-components/src/page_components/listing/listing_sidebar/Apply.stories.tsx

```diff
@@ -0,0 +1,27 @@
+import * as React from "react"
+import { storiesOf } from "@storybook/react"
+import Apply from "./Apply"
+import Archer from "@bloom-housing/listings-service/listings/archer.json"
+
+const listing = Object.assign({}, Archer)
+
+storiesOf("Listing|Sidebar Apply", module).add("hard application deadline", () => {
+  listing.applicationDueDate = "2021-11-30T15:22:57.000-07:00"
+  listing.acceptsPostmarkedApplications = false
+
+  /* eslint-disable @typescript-eslint/ban-ts-ignore */
```

Comment:
I had to disable TS checking here because otherwise _tons_ of errors were coming up related to the JSON data for listings/units. I initially started to try to fix those errors, but then ran into errors elsewhere in the listing service and website, so I think that should be filed as a separate issue.